source/math/IntBox2D.ooc

```diff
@@ -84,6 +84,13 @@ IntBox2D: cover {
 		height := Int maximum~two(0, Int maximum(this bottom, other bottom) - top)
 		This new(left, top, width, height)
 	}
+	union: func ~point (point: IntPoint2D) -> This {
+		left := Int minimum~two(this left, point x)
+		top := Int minimum~two(this top, point y)
+		width := Int maximum~two(0, Int maximum~two(this right, point x) - left)
+		height := Int maximum~two(0, Int maximum~two(this bottom, point y) - top)
+		This new(left, top, width, height)
```

Comment:
Probably... but seriously... shall I spend the time for rewriting and testing it? I thought we are in a "merging pull request" sprint. This change does not fix a bug, it _only_ makes the code more compact. It does not even increase the readability. `union()` is done everywhere like this. If this is a problem, please make an issue.